src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -418,28 +412,21 @@ def _get_install_info(cmd,
         cred_client = cf_acr_token_credentials(cmd.cli_ctx)
         poller = acr_token_credential_generate(
             cmd, cred_client, registry_name, sync_token_name,
-            password1=True, password2=True, resource_group_name=resource_group_name)
+            password1=True, password2=False, resource_group_name=resource_group_name)
         credentials = LongRunningOperation(cmd.cli_ctx)(poller)
         sync_username = credentials.username
-        sync_password = {
-            "password1": credentials.passwords[0].value,
-            "password2": credentials.passwords[1].value
-        }
+        sync_password = credentials.passwords[0].value
         logger.warning('Please store your generated credentials safely.')
     else:
         sync_username = sync_token_name
         sync_password = "<sync token password>"
+        connection_string_sync_password = sync_password
 
-    logger.warning("Value 'ACR_SYNC_TOKEN_USERNAME' has been deprecated and will be removed in a future release."
-                   " Use 'ACR_SYNC_TOKEN_NAME' instead.")
+    connection_string = "ConnectedRegistryName=%s;" % connected_registry_name + \
+        "SyncTokenName=%s;SyncTokenPassword=%s;" % (sync_username, connection_string_sync_password) + \
+        "ParentGatewayEndpoint=%s;ParentEndpointProtocol=https" % parent_gateway_endpoint
```

Comment:
ParentEndpointProtocol can be "https" or "http" if parent is connected registry.